app/api/scraping/tiktok/route.ts

```
@@ -221,209 +225,236 @@ export async function POST(req: Request) {
                     error: 'Job exceeded maximum allowed time'
-                });
+                }, 200, requestId);
             }
         }
 
-        // TEMPORARILY DISABLED FOR TESTING - Comprobar si el job está en procesamiento pero no se ha actualizado en los últimos 5 minutos
-        // Esto indica que posiblemente el proceso se interrumpió
+        // Check for stalled jobs (temporarily disabled)
         if (false && (job.status === 'processing' || job.status === 'pending') && 
             job.updatedAt && 
             (new Date().getTime() - new Date(job.updatedAt).getTime()) > 5 * 60 * 1000) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)